crates/forge/src/test_case_summary.rs

```diff
@@ -34,7 +34,12 @@ pub enum TestCaseSummary {
         /// Statistic for fuzzing test
         fuzzing_statistic: Option<FuzzingStatistics>,
     },
-    /// Test case skipped (did not run)
+    /// Test case ignored due to `#[ignored]` attribute or `--ignored` flag
+    Ignored {
+        /// Name of the test case
+        name: String,
+    },
```

Comment:
I think it should be a sub-element of Skipped.

Reply:
Discussed offline, leaving it for now

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/926